src/components/Navbar.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const Navbar = () => (
+  <>
+    <nav>
+      <ul>
+        <li>Home</li>
+        <li>BookList</li>
+        <li>Category</li>
+      </ul>
+    </nav>
+  </>
```

Comment:
Great work creating a separate component for your navbar.

- [ ] It will be great to have` <Link>`s in the navbar which leads to the booklist and categories locations. This is a requirement for this project before it can get approved.